python/pyarrow/_compute.pyx

```diff
@@ -1326,6 +1781,25 @@ cdef class _QuantileOptions(FunctionOptions):
 
 
 class QuantileOptions(_QuantileOptions):
+    __doc__ = f"""
+    Options for the `quantile` function.
+
+    Parameters
+    ----------
+    q : double or sequence of double, default 0.5
+        Quantiles to compute. All values must be in [0, 1].
+    interpolation : str, default "linear"
+        How to break ties between competing data points for a given quantile.
+        Accepted values are:
+        - "linear": compute an interpolation
```

Comment:
```suggestion

        - "linear": compute an interpolation
```

(it's ugly, but I seem to remember this is needed for proper rendering in the online docstring pages -> lists need to be surrounded by empty line in rst, but this is only within the description, so no white line is needed at the end of the description)

Reply:
Good point :-)